act-framework.bs

```diff
@@ -131,7 +131,18 @@ Selector syntax depends on the input type. When the input data is an HTML docume
 Test Cases {#test-proc-cases}
 ---------------------------------------
 
-Editor note: This section describes how rules are broken down into one or more test cases. Each test case gives some result that, when combined, provides the outcome of the rule. Additionally this section describes how rule authors should write test cases, and the mechanism of combining their outcomes.
+Each rule contains one or more testcases written in plain language. Each testcase MUST provide the following:
```

Comment:
From today's meeting we are going with test cases. I will check the document, update and merge the PR. Thanks, Moe